pymess/backend/dialer/__init__.py

```diff
@@ -41,7 +41,8 @@ def get_batch_max_seconds_to_send(self):
     def get_retry_sending(self):
         return settings.DIALER_RETRY_SENDING and self.is_turned_on_batch_sending()
 
-    def create_message(self, recipient, content, related_objects, tag, template, is_autodialer=True, **kwargs):
+    def create_message(self, recipient, content, related_objects, tag, template, is_autodialer=True,
+                       priority=settings.DEFAULT_MESSAGE_PRIORITY, **kwargs):
```

Comment:
I made it explicit in all `create_message` methods because most of the time Kwargs are inserted into `extra_data` field.

Reply:
You should then make it explicit also in `BaseBackend`.

Reply:
right. I have fixed that.